client/components/screen-options-tab/style.scss

```diff
@@ -0,0 +1,133 @@
+@import '@automattic/typography/styles/variables';
+@import '@wordpress/base-styles/variables';
+@import '@wordpress/base-styles/colors';
+
+/*
+	Screen Options Tab
+ */
+
+$screen-options-icon-border-x: 5px;
+$screen-options-icon-border-y: 7px;
+
+.screen-options-tab {
+	position: fixed;
+	top: var( --masterbar-height );
+	right: 20px;
+	z-index: z-index( 'root', '.screen-options-tab' );
+}
+
+.screen-options-tab__button {
+	border: 1px solid var( --color-neutral-5 );
+	border-radius: 0 0 4px 4px; /* stylelint-disable-line */
+	background-color: #ffffff;
+	font-size: $default-font-size;
+	padding: 3px 16px;
+	color: $gray-700;
+	cursor: pointer;
+	line-height: 1.7;
+
+	&:hover,
+	&:focus-visible {
+		color: var( --color-neutral-70 );
+
+		.screen-options-tab__icon {
+			border-top-color: var( --color-neutral-70 );
+			border-bottom-color: var( --color-neutral-70 );
+		}
+	}
+
+	&:focus-visible {
+		border-color: var( --studio-blue-50 );
+		box-shadow: 0 0 3px var( --studio-blue-10 );
```

Comment:
Do the Studio Blue colors work for all color schemes or are they specific to the Default / Classic Bright ones?

Reply:
Nope you're right, different focus colors for different themes. Made an update.